iac-common/src/test/java/org/sonar/iac/common/AbstractTestCommonSyntaxToken.java

```diff
@@ -17,39 +17,33 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-package org.sonar.iac.docker.tree.impl;
+package org.sonar.iac.common;
 
-import java.util.ArrayList;
+import java.util.Collections;
 import java.util.List;
+import org.sonar.iac.common.api.tree.Comment;
+import org.sonar.iac.common.api.tree.CommonSyntaxToken;
 import org.sonar.iac.common.api.tree.Tree;
-import org.sonar.iac.docker.tree.api.SeparatedList;
-import org.sonar.iac.docker.tree.api.SyntaxToken;
+import org.sonar.iac.common.api.tree.impl.TextRange;
 
-public class SeparatedListImpl<T extends Tree> implements SeparatedList<T> {
-
-  private final List<T> elements;
-  private final List<SyntaxToken> separators;
-
-  public SeparatedListImpl(List<T> elements, List<SyntaxToken> separators) {
-    this.elements = elements;
-    this.separators = separators;
+public abstract class AbstractTestCommonSyntaxToken implements CommonSyntaxToken {
+  @Override
+  public List<Comment> comments() {
+    return Collections.emptyList();
   }
 
   @Override
-  public List<T> elements() {
-    return elements;
+  public TextRange textRange() {
+    return null;
   }
 
   @Override
-  public List<SyntaxToken> separators() {
-    return separators;
+  public String value() {
+    return "";
   }
```

Comment:
I would remove those methods, to me they are invalid and must be overridden in any case, so by removing them from there we ensure that any implementation done later does not forget them.

Reply:
I based my implementation on the TestTree, but I agree and will change it.